src/libsync/clientsideencryption.cpp

```diff
@@ -800,6 +802,32 @@ void ClientSideEncryption::fetchFromKeyChain(const AccountPtr &account)
     job->start();
 }
 
+bool ClientSideEncryption::checkPrivateKey(const AccountPtr &account) const
```

Comment:
If I understand correct, this function validates the private key? Maybe we can give it a little bit more expressive name that describes what it checks.

Reply:
sure
like `checkPublicKeyMatchesPrivateKey` ?

Reply:
It actually checks the public key. You kind of know your private key is valid since you have to decrypt it properly.

Reply:
@mgallien sound good

Reply:
I was suggested also `checkPublicKeyValidity` and that sounds exactly what is being done. I will adopt that